api/API.md

```diff
@@ -54,6 +54,8 @@ In general, there is one directory for each API endpoint:
 )
 * `metrics`: The ZMetricMsg message sent from Device to Controller periodically to report on resource usage etc.
 * `logs`: The LogBundle message sent from Device to Controller containing internal device logs.
+* `certs`: The ZEdgeNodeCert message is sent from Device to Controller to publish public key certificates used by Device. The ZControllerCert message is sent from Controller to Device, and contains the list of certificates used by Controller.
```

Comment:
Hey @cshari-zededa either here or in the following section you need to provide workflow for how the state of  certificates gets synchronized between the cloud and the device. IOW:
   * device generates a certificate
   * device publishes certificate to the cloud via  ZEdgeNodeCert message
  
At this point, does the cloud append this new cert to the overall list of certs? IOW, if I just do a GET on /certs what will I see?

Also, how can I reset that list of ceritifacts if cloud always appends?

Reply:
Cloud always overwrites its list with the list coming in from each ZEdgeNodeCert message. Similarly, each ZControllerCert message replaces the current list on EVE with the new list of certificates. If we want to reset the list, we need to send an empty list.

Reply:
Ok. That makes sense for now. Please add this to the documentation @cshari-zededa

Reply:
updated API.md